src/test/java/com/fullteaching/e2e/no_elastest/functional/test/media/FullTeachingEndToEndRESTTests.java

```diff
@@ -37,6 +38,9 @@
 import java.time.format.DateTimeFormatter;
 import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Is this used here?

Reply:
The assertNotNull is used in the enterCourseandNavigate tab method to check that the course is found.